mir_eval/separation.py

```diff
@@ -114,6 +108,12 @@ def validate(reference_sources, estimated_sources):
                                                 MAX_SOURCES))
 
 
+def _any_source_silent(sources):
+    """Returns true if the parameter sources has any silent first dimensions"""
+    return np.any(np.all(np.sum(
+        sources, axis=np.arange(2, sources.ndim)) == 0, axis=1))
```

Comment:
> np.arange(2,2) returns [], but the axis arg to np.sum requires an int or a
tuple (not a list).

This is what I consider weird and disappointing.  I don't really see a reason they can accept tuple but not a list (or ndarray, or other iterable).

> axis=tuple(np.arange(...)) will work, I think.

`tuple(range(...))` is better.